stack.yaml

```diff
@@ -25,13 +25,13 @@ extra-deps:
 - servant-auth-server-0.4.0.0
 
 - git: https://github.com/serokell/auth-data-structures.git
-  commit: f9a0cd66f443d7b0e04a3c8ae8d475f8344b823b
+  commit: 9167add1907456f62f6ed633228198875fd39207
 
 - git: https://github.com/serokell/direct-sqlite.git
   commit: 333a2b2cf113e115ea2ec450dc706d3f47ee54e4
 
 - git: https://github.com/serokell/blockchain-util.git
-  commit: 4d8e26e7f1f00449082444ebcdf8ecd6bf422843 # dscp/release-alpha
+  commit: f4459159a6f2ddd1fea0452aa92528bff6eb000b # volhovm/dscp193
```

Comment:
Must be merged into the release branch before.

Reply:
I'm neither author nor directly related to any of these branches. Can you do it?